common/chat/contacts.ts

```diff
@@ -94,6 +95,7 @@ const getMatchContactsForUser = async (user: User): Promise<MatchContactStudent[
 };
 const getInstructorsForPupilSubcourses = async (pupil: User): Promise<SubcourseContactStudent[]> => {
     assert(pupil.pupilId, 'Pupil must have an pupilId');
+    const today = moment().toISOString();
```

Comment:
So one cannot contact participants while the last lecture is running?

Reply:
no it should be possible. Can you help me with that? @Jonasdoubleyou

Reply:
Just add a day to now, that should be a good enough estimate

Reply:
> Just add a day to now, that should be a good enough estimate

@Jonasdoubleyou but what if I have a subcourse with only one lecture and this lecture is today? Than +1 day wont work.

Reply:
Why?

Reply:
`lecture.start === today` **is not** `>= today + 1 day`

Reply:
Ah true, meant the other direction, >= yesterday